Source/pfile.cpp

```diff
@@ -64,6 +64,30 @@ std::string GetSavePath(uint32_t saveNum)
 	return path;
 }
 
+std::string GetStashSavePath()
+{
+	std::string path = paths::PrefPath();
+	const char *ext = ".sv";
+	if (gbIsHellfire)
+		ext = ".hsv";
+
+	if (gbIsSpawn) {
+		if (!gbIsMultiplayer) {
+			path.append("spawn_spash");
+		} else {
+			path.append("share_stash");
+		}
+	} else {
+		if (!gbIsMultiplayer) {
+			path.append("single_stash");
+		} else {
+			path.append("multi_stash");
+		}
+	}
+	path.append(ext);
+	return path;

```

Comment:
```suggestion
    `return "stash.sv";`
```

Create multiple files insides of the archive instead.

Reply:
I have to disagree with this idea; For the sake of moving files around, this might not be ideal for some players. Perhaps they would want to delete their Hellfire Stash for some reason, but maintain their Diablo stash. I think this could be comparable to packing all Diablo, Shareware, Hellfire SP and MP files into one big file.